flutter/lib/src/widgets_binding_observer.dart

```diff
@@ -41,14 +41,14 @@ class SentryWidgetsBindingObserver with WidgetsBindingObserver {
   ///   - [WidgetsBindingObserver](https://api.flutter.dev/flutter/widgets/WidgetsBindingObserver-class.html)
   @override
   void didChangeAppLifecycleState(AppLifecycleState state) {
-    if (!_options.enableLifecycleBreadcrumbs) {
+    if (!_options.enableAppLifecycleBreadcrumbs) {
       return;
     }
-    // According to
+    // References:
     // https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/
-    // this is of the type navigation.
+    // https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/LifecycleWatcher.java#L119-L128
```

Comment:
a tip when using links to other repos is using a permalink (that includes a SHA) so if we ref. the code and the lines change, it'd still work.